.github/ISSUE_TEMPLATE/new-feature.md

```diff
@@ -0,0 +1,31 @@
+---
+name: ✨ New Feature 
+about: Suggest a new idea for the project.
+labels: 'Type: Feature'
+title: "✨ "
+---
+
+<!-- **Tip:** Delete parts that are not relevent -->
+
+### 🩸 Pain
+<!-- Explain the pain you are experiencing -->
+
+### 💡 Suggested Solution
+<!-- Describe the intended solution -->
+
+### 💼 Tasks 
+<!-- Add GitHub tasks in a measurable, check-box manner -->
+- [ ] ... 
+- [ ] ... 
+
+### ✅ Acceptance Criteria 
+<!-- Add the "contract" that defines the requirements for the GitHub issue to be completed as per the team's agreement -->
+
+### ➕ More Information
+<!-- Add any other context here -->
```

Comment:
Expand a bit more on the info here.
<!-- Would love to have more info like device type, OS etc -->